Sources/GDSAnalytics/Screens/NamedScreen.swift

```diff
@@ -1,3 +1,5 @@
 public protocol NamedScreen {
```

Comment:
this is a huge breaking change across our codebases, can we find a better way that's backward compatible?

Reply:
Same as above, but sure can revisit this

Reply:
It's an open-closed violation - our module is not open to extension if we add this here.